examples/producer.md

```diff
@@ -47,7 +47,10 @@ producer.on('ready', function(arg) {
     var key = "key-"+i;
     // if partition is set to -1, librdkafka will use the default partitioner
     var partition = -1;
-    producer.produce(topicName, partition, value, key);
+    var headers = [
+      { header: "header value" }
+    ]
+    producer.produce(topicName, partition, value, key, new Date(), "". headers);
```

Comment:
Why `"".` in front of the headers collection?  Looks like this should be `,` instead of `.`.

Reply:
Yeah I think this is a typo here. Would probably be good to get that fixed.